v2.0/create-sequence.md

```diff
@@ -105,14 +130,19 @@ Insert a few records to see the sequence.
 +----+----------+--------------------+
 ~~~
 
-<!-- ### View the Current Value of a Sequence
+### View the Current Value of a Sequence
 
 {% include copy-clipboard.html %}
 ~~~ sql
 > SELECT * FROM customer_seq;
 ~~~
 ~~~
-~~~ -->
++------------+---------+-----------+
+| last_value | log_cnt | is_called |
++------------+---------+-----------+
+|          3 |       0 |   true    |
++------------+---------+-----------+
+~~~
 
```

Comment:
It'd be helpful to explain the returned fields. 

Also, we could add:

> If a value has been obtained from the sequence in the current session, you can also use the `curval()` function to get that most recently obtained value:

Followed by:

```
> SELECT curval('customer_seq');
```

Reply:
Done - added `currval()`.

@vilterp - can you help me understand the returned fields for `SELECT* FROM customer_seq`?  On the same note, from what I understand, our use of `is_called` in `setval('seq_name', value, is_called)`differs from Postgres and we need to call that out. I'm not sure I understand what's different. Can you explain?

Reply:
I actually wrote an issue about that: #21564. Not sure how much of that you want to document here vs a known limitation; it's kind of awkward. Let me know what you think.